<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>work_counter</class>
 <widget class="QWidget" name="work_counter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="c_elecrt">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="have_electr">
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>27</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="val_elect">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>70</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="on_electr">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>27</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="norm_elect">
   <property name="geometry">
    <rect>
     <x>526</x>
     <y>27</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="val_electr_tar">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>70</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="speed_electr">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="speed_electr_tar">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>69</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="last_elecrt">
   <property name="geometry">
    <rect>
     <x>756</x>
     <y>70</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="electr_last_tar">
   <property name="geometry">
    <rect>
     <x>831</x>
     <y>70</y>
     <width>110</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="c_gas">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>95</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gas_last_tar">
   <property name="geometry">
    <rect>
     <x>831</x>
     <y>145</y>
     <width>110</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="last_gas">
   <property name="geometry">
    <rect>
     <x>756</x>
     <y>145</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="speed_gas">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>145</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="speed_gas_tar">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>144</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="val_gas">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>145</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="val_gas_tar">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>145</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="norm_gas">
   <property name="geometry">
    <rect>
     <x>526</x>
     <y>99</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="on_gas">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>100</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="c_water">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>175</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="water_last_tar">
   <property name="geometry">
    <rect>
     <x>831</x>
     <y>225</y>
     <width>110</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="last_water">
   <property name="geometry">
    <rect>
     <x>756</x>
     <y>225</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="speed_water">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>225</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="speed_water_tar">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>224</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="val_water">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>225</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="val_water_tar">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>225</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="norm_water">
   <property name="geometry">
    <rect>
     <x>526</x>
     <y>179</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="on_water">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>180</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>476</x>
     <y>354</y>
     <width>144</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="have_gas">
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>100</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="have_water">
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>180</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="warning">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>941</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="on_true">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>299</y>
     <width>146</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="on_false">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>299</y>
     <width>146</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="have_true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>299</y>
     <width>146</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="have_false">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>299</y>
     <width>146</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="norm_true">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>299</y>
     <width>146</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="norm_false">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>299</y>
     <width>146</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1087</width>
     <height>399</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pich.qrc">:/picha/Pichures/work_counter.png</pixmap>
   </property>
  </widget>
  <zorder>Background</zorder>
  <zorder>c_elecrt</zorder>
  <zorder>have_electr</zorder>
  <zorder>val_elect</zorder>
  <zorder>on_electr</zorder>
  <zorder>norm_elect</zorder>
  <zorder>val_electr_tar</zorder>
  <zorder>speed_electr</zorder>
  <zorder>speed_electr_tar</zorder>
  <zorder>last_elecrt</zorder>
  <zorder>electr_last_tar</zorder>
  <zorder>c_gas</zorder>
  <zorder>gas_last_tar</zorder>
  <zorder>last_gas</zorder>
  <zorder>speed_gas</zorder>
  <zorder>speed_gas_tar</zorder>
  <zorder>val_gas</zorder>
  <zorder>val_gas_tar</zorder>
  <zorder>norm_gas</zorder>
  <zorder>on_gas</zorder>
  <zorder>c_water</zorder>
  <zorder>water_last_tar</zorder>
  <zorder>last_water</zorder>
  <zorder>speed_water</zorder>
  <zorder>speed_water_tar</zorder>
  <zorder>val_water</zorder>
  <zorder>val_water_tar</zorder>
  <zorder>norm_water</zorder>
  <zorder>on_water</zorder>
  <zorder>pushButton</zorder>
  <zorder>have_gas</zorder>
  <zorder>have_water</zorder>
  <zorder>warning</zorder>
  <zorder>on_true</zorder>
  <zorder>on_false</zorder>
  <zorder>have_true</zorder>
  <zorder>have_false</zorder>
  <zorder>norm_true</zorder>
  <zorder>norm_false</zorder>
 </widget>
 <resources>
  <include location="pich.qrc"/>
 </resources>
 <connections/>
</ui>
